properties.go

```diff
@@ -647,6 +622,33 @@ func (p *Properties) WriteComment(w io.Writer, prefix string, enc Encoding) (n i
 	return
 }
 
+// ToMap returns a copy of the properties as a map - useful when users of library are themselves
```

Comment:
Pls remove the ` - useful ...` part since you don't know what users will want to use this for.